lib/anyway/loaders/doppler.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'uri'
+require 'net/http'
+require "json"
+
+module Anyway
+  using RubyNext
+
+  module Loaders
+    class Doppler < Base
+      DOPPLER_REQUEST_ERROR = Class.new(StandardError)
+      DOPPLER_JSON_FORMAT_URL = "https://api.doppler.com/v3/configs/config/secrets/download"
+
+      def call(**_options)
```

Comment:
We can probably use forwarding here to not care about the method signature 

```suggestion
      def call(...)
```

Reply:
thanks!

but now here I set

```
(env_prefix:, **_options)
```

because, I use env_prefix now for Anyway::Env